### ISSUE TITLE
data-footer-formatter 1.13.1 not work

### ISSUE THREAD

**Issue by dabcorp** [wenzhixin/bootstrap-table#4095]
Hello,

After migrate to version 1.13.1 , data footer formatter not work :

HTML :

`<th data-field="Montant" data-footer-formatter="totalligne" data-filter-control="input" data-sortable="true" data-align="center">Montant HT</th>`

JS : 

```
function totalligne(data) {
		var totalligne = 0;
		$.each(data, function (i, row) {
			totalligne += +parseFloat(row.PrixLigne.replace(",", ".")).toFixed(2);			
		});
		$("#MontantHT,#TOTALHT").html(totalligne.toFixed(2)+' €');
};
```

Thx

**Comment by trickdaddylatty** [comment]
trickdaddylatty72

**Comment by dabcorp** [comment]
??

**Comment by djhvscf** [comment]
Please provide a jsfiddle for this issue

**Comment by wenzhixin** [comment]
Fixed in v1.13.5.
https://examples.bootstrap-table.com/#options/show-footer.html
